graylog2-web-interface/src/views/components/aggregationwizard/sort/SortConfiguration.tsx

```diff
@@ -61,6 +61,7 @@ const Sort = React.memo(({ index }: Props) => {
   const hasGroupings = otherSorts.find((s) => s.type === 'groupBy');
 
   const metricsOptions: Array<OptionValue> = hasGroupings ? [] : metrics.map(formatSeries)
+    .filter((metric) => !(metric.label.startsWith('latest(')))
```

Comment:
Let's filter `metrics` before formatting them with `formatSeries`. It is a little bit cleaner, because each object still has an id, the `function` attribute.